### PR TITLE
406-fibo-finance

### PR DESCRIPTION
**Pull request by fkim7**

### CODE REVIEW

**Review by BenjaminMoe — CHANGES_REQUESTED** [review]
Looks like there are some errors with the CI. https://github.com/w3c-ccg/traceability-vocab/actions/runs/3593988961/jobs/6118985939

Edit: we can help with tooling if you have any questions.

File: docs/openapi/components/schemas/common/BankAccount.yml
Comment (TallTed) [inline docs/openapi/components/schemas/common/BankAccount.yml:16]:
```suggestion
    description: Business Identifier Code — ISO 9362 identification of the financial institution’s branch.
```

File: docs/openapi/components/schemas/common/BankAccount.yml
Comment (TallTed) [inline docs/openapi/components/schemas/common/BankAccount.yml:22]:
```suggestion
    description: International Bank Account Number — ISO 13616 identification of bank accounts across national borders.
```

**Comment by nissimsan** [comment]
@fkim7 , we're discussing this on the call. We are happy to help get the issue resolved. You are also very welcome to join the weekly call. 
COntent looks good - we'd love to merge this.

**Comment by fkim7** [comment]
Yes I saw the invalid IRI- Where can I put the content of https://knox-networks.github.io/schema/bankaccount/v1.jsonld ?

**Review by BenjaminMoe — APPROVED** [review]
Looks like this is from a fork. I'd be okay with merging this if we make an issue to address an CI issue after merge.

**Comment by fkim7** [comment]
> > Yes I saw the invalid IRI- Where can I put the content of https://knox-networks.github.io/schema/bankaccount/v1.jsonld ?
> 
> You can put the content of the `context` json, in here: https://github.com/w3c-ccg/traceability-vocab/blob/main/packages/traceability-schemas/services/contexts.js#L23

Thank you! Updated context to be under https://w3id.org/traceability/v1 and updated the file name under /credential

Can someone review and approve & merge if all looks good?

**Comment by BenjaminMoe** [comment]
> Can someone review and approve & merge if all looks good?

@fkim7, we'll review it on the call in an about an hour which you're welcome to come if you want to attend: https://github.com/w3c-ccg/traceability-vocab#meetings

**Comment by nissimsan** [comment]
Merging, @BenjaminMoe , please fix the CI issues subsequently

**Pull request merged**
